components/python/salt/salt-PYVER.p5m

```diff
@@ -0,0 +1,1637 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2018 Till Wegmueller
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)-$(PYV)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value='Portable, distributed, remote execution and configuration management system'
+set name=info.classification value='org.opensolaris.category.2008:System/Administration and Configuration'
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='Apache License 2.0'
+
+# Tornado does a lot of try/import/except, making pkgdepend unhapyy
+<transform path=usr/lib/python$(PYVER)/vendor-packages/salt/.*\.py -> default pkg.depend.bypass-generate .* >
+
+# force a dependency on the Python runtime
+depend type=require fmri=__TBD pkg.debug.depend.file=python$(PYVER) \
+    pkg.debug.depend.path=usr/bin
+
+# dependencies
+depend type=require fmri=library/python/msgpack-python-$(PYV)
+depend type=require fmri=library/python/pyyaml-$(PYV)
+depend type=require fmri=library/python/jinja2-$(PYV)
+depend type=require fmri=library/python/markupsafe-$(PYV)
+depend type=require fmri=library/python/requests-$(PYV)
+depend type=require fmri=library/python/tornado-$(PYV)
+depend type=require fmri=library/python/pyzmq-$(PYV)
+depend type=require fmri=library/python/cryptography-$(PYV)
+depend type=require fmri=library/python/futures-$(PYV)
+
+# SMF Manifest and distribution directories
+set name=org.opensolaris.smf.fmri value=svc:/application/salt:minion value=svc:/application/salt:master
+dir  path=etc/salt owner=root group=root mode=0755
+dir  path=var/log/salt owner=root group=root mode=0755
+file files/salt.xml path=lib/svc/manifest/application/salt.xml
+
+
+file path=usr/bin/salt
```

Comment:
If manifest is versioned, better move usr/bin/* to usr/bin/*-2.7/3.4/3.5 and make mediated symlinks to them

Reply:
mediated as implementation or version?

Reply:
Using version mediation. and setting version to 2.7